bundles/org.openhab.binding.tibber/src/main/java/org/openhab/binding/tibber/internal/handler/TibberHandler.java

```diff
@@ -0,0 +1,471 @@
+                }
+                if (myObject.has("accumulatedCost")) {
+                    updateChannel(LIVE_ACCUMULATEDCOST, myObject.get("accumulatedCost").toString());
+                }
+                if (myObject.has("currency")) {
+                    updateState(LIVE_CURRENCY, new StringType(myObject.get("currency").toString()));
+                }
+                if (myObject.has("minPower")) {
+                    updateChannel(LIVE_MINPOWER, myObject.get("minPower").toString());
+                }
+                if (myObject.has("averagePower")) {
+                    updateChannel(LIVE_AVERAGEPOWER, myObject.get("averagePower").toString());
+                }
+                if (myObject.has("maxPower")) {
+                    updateChannel(LIVE_MAXPOWER, myObject.get("maxPower").toString());
+                }
+                if (myObject.has("voltagePhase1")) {
+                    updateChannel(LIVE_VOLTAGE1, myObject.get("voltagePhase1").toString());
+                }
+                if (myObject.has("voltagePhase2")) {
+                    updateChannel(LIVE_VOLTAGE2, myObject.get("voltagePhase2").toString());
+                }
+                if (myObject.has("voltagePhase3")) {
+                    updateChannel(LIVE_VOLTAGE3, myObject.get("voltagePhase3").toString());
+                }
+                if (myObject.has("currentPhase1")) {
+                    updateChannel(LIVE_CURRENT1, myObject.get("currentPhase1").toString());
+                }
+                if (myObject.has("currentPhase2")) {
+                    updateChannel(LIVE_CURRENT2, myObject.get("currentPhase2").toString());
+                }
+                if (myObject.has("currentPhase3")) {
+                    updateChannel(LIVE_CURRENT3, myObject.get("currentPhase3").toString());
+                }
+                if (myObject.has("powerProduction")) {
+                    updateChannel(LIVE_POWERPRODUCTION, myObject.get("powerProduction").toString());
+                }
+                if (myObject.has("accumulatedProduction")) {
+                    updateChannel(LIVE_ACCUMULATEDPRODUCTION, myObject.get("accumulatedProduction").toString());
+                }
+                if (myObject.has("minPowerProduction")) {
+                    updateChannel(LIVE_MINPOWERPRODUCTION, myObject.get("minPowerProduction").toString());
+                }
+                if (myObject.has("maxPowerProduction")) {
+                    updateChannel(LIVE_MAXPOWERPRODUCTION, myObject.get("maxPowerProduction").toString());
+                }
+            } else {
+                logger.debug("Unknown live response from Tibber");
+            }
+        }
+
+        @SuppressWarnings("null")
+        private void sendMessage(String message) throws IOException {
+            logger.debug("Send message: {}", message);
+            session.getRemote().sendString(message);
```

Comment:
This should resolve the warning so you can remove the suppression.
```suggestion
            Session session = TibberHandler.this.session;
            if(session != null){
                RemoteEndpoint remote = getRemote();
                if(remote != null){
                    remote.sendString(message);
                }
            }
```

Reply:
Suppression removed, although needed to adjust code as `RemoteEndpoint remote = getRemote();` was causing error (getRemote() is undefined for type).

Working code:
`Session session = TibberHandler.this.session;
            if (session != null) {
                session.getRemote().sendString(message);
            }`